client/src/i18n/locales/pl.json

```diff
@@ -126,5 +126,23 @@
   },
   "Modal": {
     "Close": "Zamknij"
+  },
+  "ServerErrors": {
+    "NotFound": "Nie znaleziono zasobu.",
+    "Unauthorized": "Użytkownik niezalogowany. Przejdź na stronę logowania, aby móc korzystać z aplikacji.",
+    "AccessDenied": "Brak dostępu do zasobu.",
+    "UnknownError": "Wystąpił błąd. Spróbuj ponownie później lub skontaktuj się z administratorem systemu.",
+    "Required": "Nie wszystkie wymagane pola zostały podane.",
+    "TooOldRepetition": "Powtórzenie błędu nie powinno być wcześniej niż dzień wcześniej.",
```

Comment:
It sounds weird :D maybe something like "powtórzenie błędu nie może być starsze niż jeden dzień"